src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerPlaybackIntTest.java

```diff
@@ -116,15 +111,6 @@ void audioRequestPlaybackShouldPlaybackFromOutboundStorageAndReturnSuccess() thr
             Set.of(JUDGE, REQUESTER, APPROVER, TRANSCRIBER, TRANSLATION_QA)
         );
 
-        AuditSearchQuery searchQuery = new AuditSearchQuery();
-        searchQuery.setCaseId(mediaRequestEntity.getHearing().getCourtCase().getId());
-        searchQuery.setFromDate(OffsetDateTime.now().minusDays(1));
-        searchQuery.setToDate(OffsetDateTime.now().plusDays(1));
-        searchQuery.setAuditActivityId(PLAYBACK_AUDIT_ACTIVITY_ID);
-
-        List<AuditEntity> auditEntities = auditService.search(searchQuery);
-        assertEquals("2", auditEntities.get(0).getCourtCase().getCaseNumber());
-        assertEquals(1, auditEntities.size());
```

Comment:
Again, this should be tested in a different way.

Reply:
Code updated for tests